src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -44,6 +37,19 @@ class SignInModal extends React.Component {
     }
   }
 
+  authVersion() {
+    return this.props.useSSOe ? 'v1' : 'v0';
+  }
```

Comment:
This could also possibly be declared in the `mapStatesToProps` and invoked as `this.props.authVersion`, but doesn't really make a difference to me.

```
const mapStateToProps = () => {
  const useSSOe = ssoe(state);

  return {
    useSSOe,
    authVersion: useSSOe ? 'v1' : 'v0',
  };
};
```

Reply:
oh cool, then I wouldn't even need the `useSSOe` function